README.md

```diff
@@ -1,19 +1,30 @@
 # Conjur API for .NET
 
-This is a *Draft* implementation of the .NET API for [V5 Conjur](https://developer.conjur.net/).
-This implementation includes an example that shows how to:
+Programmatic .NET access to the [Conjur API](https://docs.conjur.org/latest/en/Content/Developer/lp_REST_API.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____0#rest-api) (for both Conjur OSS and Enterprise/DAP versions).
+This .NET SDK allows developers to build new apps in .NET that communicate with Conjur by
+invoking our Conjur API to perform operations on stored data (add, retrieve, etc)
 
-    - Authenticate
-    - Load Policy
-    - Check permissions to get the value of a variable
-    - Get the value of a variable
-    - Use a Host Factory token to create a new Host and get an apiKey to use with Conjur
+## Table of Contents
+
+- [Requirements](#requirements)
+- [Building](#building)
+- [Methods](#methods)
+- [Example](#example)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Requirements
+
+Ensure your Conjur/DAP version use the [V5 API](https://docs.conjur.org/latest/en/Content/Developer/lp_REST_API.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____0#rest-api).
```

Comment:
```suggestion
- DAP v10+ or Conjur OSS v1+
```
Are there any requirements on the language version that they use?

Reply:
I'll add these suggestions. I added the .NET language version requirement to [CONTRIBUTING](https://github.com/cyberark/conjur-api-dotnet/blob/community-repo-standards/CONTRIBUTING.md)

Reply:
but if I'm an end user, will this client only work with specific versions of .net?

Reply:
In all honesty, I don't know. I only know that C#7 in the actual project breaks the test builds. I'm not sure for incorporating this into another project.

@uCatu do you know anything about this?

Reply:
@sashaCher 
Belive that there will be a minimum version of .net support from and above

Reply:
Hey guys, I moved this to a separate issue(https://github.com/cyberark/conjur-api-dotnet/issues/62), so this PR could be merged in the meantime. Could we please move the conversation over there?

Reply:
We should distinct between language support means with which toolset it's possible to compile this code (in our case the issue is that VS 2017 toolset needed in order to compile C#7 code) and between runtime support means what's minimal framework (mono, etc.) version that can run this code (it should be at least .net framework 4.5.2 according our own needs)